tests/test_blocks.py

```diff
@@ -72,7 +57,6 @@ def exampleMethod_(self, block: objc_block):
         result = instance.blockExample()
         self.assertEqual(result, 9)
 
-    @m1_block_failure
     def test_block_delegate_auto_struct(self):
```

Comment:
How about adding a test case for with a named (non-anonymous) structure as argument as well?

Reply:
Done. I've added a test of a mismatched structure as well - however, as Python exceptions aren't propagated back into the ObjC runtime (and then back out into Python), the test is a little weird; but I've annotated the weirdness.